controllers/accounts.js

```diff
@@ -1,104 +1,68 @@
 const accountModel = require('../models/accounts')
 
 function signup(req, res, next) {
-
```

Comment:
the `petname` was not updated in the seed/migration files. front-end also uses `displayname`. will remove the other stuff though